app/components/AnnounceBanner.tsx

```diff
@@ -84,3 +78,15 @@ export default function AnnounceBanner({
     </section>
   )
 }
+
+export function AnnouncementEvent(event: AnnouncedEvents) {
```

Comment:
```suggestion
export function AnnouncementEvent(props: AnnouncedEvents) {
```

Also just inline the type here in the function signature.